app/rtc_s1.py

```diff
@@ -262,11 +284,11 @@ def calculate_layover_shadow_mask(burst_in: Sentinel1BurstSlc,
         Iteration threshold for geo2rdr
     numiter_geo2rdr: int
         Number of max. iteration for geo2rdr object
-    nlooks_az: int
-        Number of looks in azimuth direction. For the calculation in coarse grid
-    nlooks_rg: int
-        Number of looks in range direction. For the calculation in coarse grid
 
+    Returns
```

Comment:
The docstring says that this function returns `layover_shadow_mask_raster`, but I cannot find what line in this function does.

Reply:
Thank you, @seongsujeong . Somehow the line was deleted in one of my updates. Thank you for catching that.